scripts/normalize-by-median.py

```diff
@@ -281,6 +284,10 @@ def main():  # pylint: disable=too-many-branches,too-many-statements
         print('loading k-mer counting table from ' + args.loadtable,
               file=sys.stderr)
         htable = khmer.load_counting_hash(args.loadtable)
+        if args.unique_kmers != 0:
+            print('Warning: You have specified a number of unique kmers'
+                  ' but are loading a precreated counting table--'
+                  'argument optimization will NOT be done.', file=sys.stderr)
```

Comment:
+1

On Tue, Jul 14, 2015, 07:00 C. Titus Brown <notifications@github.com> wrote:

> In scripts/normalize-by-median.py
> <https://github.com/dib-lab/khmer/pull/1126#discussion_r34570329>:
>
> > @@ -281,6 +284,10 @@ def main():  # pylint: disable=too-many-branches,too-many-statements
> >          print('loading k-mer counting table from ' + args.loadtable,
> >                file=sys.stderr)
> >          htable = khmer.load_counting_hash(args.
>
> loadtable)
> > +        if args.unique_kmers != 0:
> > +            print('Warning: You have specified a number of unique kmers'
> > +                  ' but are loading a precreated counting table--'
> > +                  'argument optimization will NOT be done.', file=sys.stderr)
>
> Is this what you were concerned about @mr-c <https://github.com/mr-c>? We
> could change this to
>
> counting table size will NOT be set automatically.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dib-lab/khmer/pull/1126/files#r34570329>.
>
-- 
Michael R. Crusoe: Programmer & Bioinformatician crusoe@ucdavis.edu
The lab for Data Intensive Biology; University of California, Davis
https://impactstory.org/MichaelRCrusoe http://twitter.com/biocrusoe